cli/create_initial_data.py

```diff
@@ -30,16 +30,18 @@
 
 path = env_service.get("DATASTORE_INITIAL_DATA_FILE")
 print(f"Loading data: {path}")
-if path.startswith("http://") or path.startswith("https://"):
-    try:
-        file = request.urlopen(path, timeout=20)
-    except URLError:
-        print(f"Timeout while fetching {path}")
-        sys.exit(1)
+if path.startswith("initial-data://"):
```

Comment:
Use `initial-data:`, so your schema is clear. The `//` is not needed ;)

Reply:
Done.